pages/extensions/amp_dependencies/__init__.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+import uuid
+from sets import Set
+
+from grow import extensions
+from grow.documents import document, static_document
+from grow.extensions import hooks
+
+class AmpDependencies(object):
+    def __init__(self, doc):
```

Comment:
For new files please use the agreed indention width of 2

Reply:
Done!